examples/node/api.js

```diff
@@ -1,23 +1,245 @@
 const express = require('express');
-const { verify } = require('@adcaptcha/node');
-const cors = require('cors');
+const dotenv = require('dotenv');
+const fs = require('fs').promises;
+const path = require('path');
+const { AdCaptchaAPIClient } = require('@adcaptcha/node');
+
+dotenv.config({ path: path.resolve(__dirname, '.env.local') });
 const app = express();
-require('dotenv').config({ path: '.env.local' });
+const client = new AdCaptchaAPIClient(process.env.ADCAPTCHA_API_KEY);
 
-app.use(cors());
-app.use(express.json());
+//  example for sites API
+async function sitesFetchAll() {
+  try {
+    const result = await client.sites.fetchAll();
+    console.log(result.data); 
+  } catch (error) {
+    console.error("Error fetching sites:", error);
+  }
+}
 
-app.post('/verifyToken', async (req, res) => {
-  const apiKey = process.env.API_KEY;
-  const token = req.body.token;
+async function sitesFetchByID() {
+  try {
+    const result = await client.sites.fetchByID("STE-01JHT1EPZ83KVZM3XZCH1YA6HD");
+    console.log(result.data); 
+  } catch (error) {
+    console.error("Error fetching sites:", error);
+  }
+}
 
+async function sitesFetchStatsForSite() {
   try {
-    const response = await verify(apiKey, token);
-    res.json(response);
+    const result = await client.sites.fetchStatsForSite("STE-01JHTK6HM50CVN2Q0BJRZT0Z3S", "2");
+    console.log(result.data.statusPerDay); 
   } catch (error) {
-    console.error(error);
-    res.status(500).json({ error: 'Failed to verify token' });
+    console.error("Error fetching sites:", error);
   }
-});
+}
+
+async function sitesCreateSite() {
+  try {
+    const result = await client.sites.createSite("Bunyo karacsonyig", "https://piedone.com");
```

Comment:
Use the adcaptcha domain in examples please. I won't raise this one again so make sure to update all examples.